.github/workflows/integration_test.yml

```diff
@@ -114,6 +114,9 @@ jobs:
         with:
           ruby-version: ${{ matrix.ruby-version }}
           bundler-cache: true
+      - name: Update RubyGems
```

Comment:
I think `setup-ruby` can do this for you, check https://github.com/ruby/setup-ruby/blob/c4fe7bd15ddbfcd5e07e47bf2f2cae90581d6091/action.yml#L15

I would do:

```
      - name: Set up Ruby
        uses: ruby/setup-ruby@v1
        with:
          ruby-version: ${{ matrix.ruby-version }}
          rubygems: latest
          bundler-cache: true
```